sds_data_manager/constructs/instrument_lambdas.py

```diff
@@ -0,0 +1,67 @@
+"""Module containing Constructs for instsrument Lambda functions"""
+from pathlib import Path
+
+from aws_cdk import Duration
+from aws_cdk import aws_lambda as lambda_
+from aws_cdk import aws_lambda_python_alpha as lambda_alpha_
+from aws_cdk import aws_s3 as s3
+from constructs import Construct
+
+
+class InstrumentLambda(Construct):
+    """Generic Construct with customizable runtime code
+    """
+
+    def __init__(self,
+                 scope: Construct,
+                 construct_id: str,
+                 sds_id:str,
+                 processing_step_name: str,
+                 archive_bucket: s3.Bucket,
+                 code_path: str or Path,
```

Comment:
since this lambda is using `code_path` to get its code, should we also add ability to pass an optional argument for lambda layers if it needs more libraries?

Reply:
Will do.

Reply:
If we do Dockerize the lambdas we need to keep in mind that that can slow down deploy time.